tests/baselines/reference/user/webpack.log

```diff
@@ -0,0 +1,12 @@
+Exit Code: 2
+Standard output:
+lib/util/cleverMerge.js(538,3): error TS2322: Type 'T' is not assignable to type 'Omit<T, P>'.
```

Comment:
*sigh* yeah, might need _full_ assignability rules for `.nameType`- having mapped types before we can ship this change, rather than just comparability rules.